### DISCUSSION TITLE
Self-nomination for Contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#145] [Ruby Team]
Hi @paketo-buildpacks/ruby-maintainers

I would like to self-nominate for the Contributor role for the ruby team. I've been making contributions to multiple repositories in the ruby project. Primarily this is the SBOM work, but I have also been picking up various isolated issues too. Becoming a contributor will enable me to contribute more effectively and will help me share the contribution responsibilities.

Thanks,
Rob

**Comment by ryanmoran** [comment]
🎉 Congrats @robdimsdale! You are now a Ruby Contributor!
